doc/source/visualizing/plots.rst

```diff
@@ -706,29 +706,98 @@ two element tuples.
    slc.set_center((0.5, 0.503))
    slc.save()
 
-Flipping the plot view axes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adjusting the plot view axes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 By default, all :class:`~yt.visualization.plot_window.PlotWindow` objects plot
 with the assumption that the eastern direction on the plot forms a right handed
 coordinate system with the ``normal`` and ``north_vector`` for the system, whether
```

Comment:
I think that this sentence is not in fact complete, because we don't mention that the normal direction is by default pointing towards the reader. Also we're mixing two paradigms (north/east VS normal direction). Overall, seems like a good opportunity to fix this.

Reply:
Working on a quick docs change --- I agree, should fix this now! But what's the correct description?

I'm inclined to just delete this sentence and  start with "There are a number of ways the orientation of the initial `PlotWindow` view can be adjusted." and not get into how the default is constructed at all, but maybe that's actually import context to keep?

Reply:
Hum... I think it's important that we have clearly documented default behaviours, so I am not in favour of deleting the existing sentence, but I would like that it also mentions that the "normal" direction points to the user _by default_. Since it's one of the things that can be adjusted, your existing transition to the next paragraph still works with my proposed addition :)

Reply:
I'll take that back. After rethinking about it I think the whole notion of "right handed" 2D plots doesn't make sense; you'd get the exact same result by assuming either right-handedness with the normal pointing towards the reader or left-handedness + normal pointing away.
Seems to me we're better off leaving "right-handedness" completely out of documention. Sorry about these back and forth, I get really confused by this at times

Reply:
> Sorry about these back and forth, I get really confused by this at times

Ya, me too :) 

So to capture the default behavior, maybe: 

"By default, all Plot Window objects plot with the normal of the plotting plane pointing to the user. There are a number of ways to adjust the initial orientation ... "